Makefile

```diff
@@ -108,8 +108,8 @@ build-local-darwin-all: build-local-darwin ## Build kusion & kcl tool chain for
 		./cmd/kcl-go
 
 	# Build kcl-openapi
-	# cd ./cmd/kcl-openapi && GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 \
-	# 	go build -o ${PWD}/_build/bundles/kusion-darwin/bin/kcl-openapi
+	cd ./cmd/kcl-openapi && GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 \
+		go build -o ${PWD}/_build/bundles/kusion-darwin/bin/kcl-openapi
 
 	# chmod +x
 	-chmod +x ./_build/bundles/kusion-darwin/bin/kusion
```

Comment:
the next line should be uncommented, too

Reply:
> the next line should be uncommented, too

fixed.